### PR TITLE
HPCC-14513 Document #OPTION('soapTraceLevel','n')

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia jamesdefabia@lexisnexis.com
@afishbeck Please review

### CODE REVIEW

**Comment by richardkchapman** [comment]
@RussWhitehead Please review

File: docs/ECLLanguageReference/ECLR_mods/Templ-OPTION.xml [hunk 1]
```diff
@@ -964,6 +964,16 @@
 
               <entry>If true, enables ASSERT checking.</entry>
             </row>
+
+            <row>
+              <entry><emphasis>soapTraceLevel</emphasis></entry>
+
+              <entry>Default: 1</entry>
+
+              <entry>The level of detail in reporting SOAPCALL information
```
Comment (afishbeck) [inline docs/ECLLanguageReference/ECLR_mods/Templ-OPTION.xml]: This actually affects HTTPCALL as well... the option name is historical from before HTTPCALL existed.

**Comment by JamesDeFabia** [comment]
@RussWhitehead Edits made, please look again

**Comment by JamesDeFabia** [comment]
I made an additional edit based upon @afishbeck 's comment

**Pull request merged**